### PR TITLE
env variable preview

### PR DESCRIPTION
**Pull request by grosser**
in our private plugin we fail when unable to resolve some values, but would be nice to be able to show a helpful message so users can see if it only affects a certain group they might not care for / gives them a complete picture of the issue instead of failing at the first one

@zendesk/runway 

### Risks
 - None

### CODE REVIEW

**Comment by grosser** [comment]
@zendesk/runway

**Comment by grosser** [comment]
@zendesk/runway

File: plugins/env/app/models/environment_variable.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ class EnvironmentVariable < ActiveRecord::Base
   validates :scope_type, inclusion: ["Environment", "DeployGroup", nil]
 
   class << self
-    def env(project, deploy_group)
+    # preview parameter can be used to not raise an error,
+    # but return a value with a helpful message
+    def env(project, deploy_group, preview: false)
```
Comment (sandlerr) [inline plugins/env/app/models/environment_variable.rb]: I'm confused, where does this method use `preview`?

Comment (grosser) [inline plugins/env/app/models/environment_variable.rb]: it does not, I'm going to use it in the consul reader to either blow up or show a warning

**Comment by sandlerr** [comment]
:+1:

**Pull request merged**